docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+services:
+  app:
+    build:
+      context: .
+    command: bundle exec rails server --bind 0.0.0.0
```

Comment:
We have to bind to `0.0.0.0` because of docker?

Reply:
I believe that's necessary so that Rails listens from any IP address (from within the container). Listening from `127.0.0.1` might not work since we have to access it from outside of the container, which has a different IP address.

Reply:
yes!